src/components/grid-aware/PartnersAndSponsorsBlock/PartnersAndSponsorsBlock.js

```diff
@@ -0,0 +1,46 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import s from "./PartnersAndSponsorsBlock.module.css";
+
+const LogoPropType = PropTypes.shape({
+  key: PropTypes.number.isRequired,
+  url: PropTypes.string.isRequired,
+  alt: PropTypes.string.isRequired,
+});
+
+const PartnersAndSponsorsBlock = ({ title, partnersAndSponsors }) => {
+  const GridAreaLeft = () => (
+    <div className={s.gridAreaLeft}>
+      <h1 className={s.title}>{title}</h1>
+    </div>
+  );
+
+  const GridAreaRight = () => (
+    <div className={s.gridAreaRight}>
+      <div className={s.logosWrapper}>
+        {partnersAndSponsors.map((logo) => (
+          <div className={s.logoContainer} key={logo.key}>
+            <img className={s.logo} src={logo.url} alt={logo.alt} />
+          </div>
+        ))}
+      </div>
+    </div>
+  );
+
+  return (
+    <div>
```

Comment:
This extra `<div>` isn't doing anything, so you can remove it. [`<section>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) is a semantic HTML tag that is visually equivalent to a `<div>` (i.e. is a block element and doesn't really have any other visible features), but it implies a certain type of semantic on the page.